src/App.vue

```diff
@@ -78,6 +84,25 @@ export default {
      */
     toggleFavourites() {
       this.showFavourites = !this.showFavourites;
+    },
+    /**
+     * Updates myFavourites array of beer ID's.
+     */
+    updateMyFavourites(id) {
+      const beerId = !!this.myFavourites.find(beer => beer === id);
+
+      beerId
```

Comment:
I find using ternary here a bit hard to read, but it's a personal opinion. If you work with others, they might find it hard to read too.

Reply:
Updated function.